scripts/missions/wotg/12_Affairs_of_State.lua

```diff
@@ -0,0 +1,138 @@
+-----------------------------------
+-- Affairs of State
+-- Wings of the Goddess Mission 12
+-----------------------------------
+-- !addmission 5 11
+-- Velda-Galda : !pos 138.631 -2.112 61.658 94
+-- Radford     : !pos -205.303 -8.000 26.874 87
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/keyitems')
+require('scripts/globals/missions')
+require('scripts/globals/interaction/mission')
+require('scripts/globals/zone')
+-----------------------------------
+
+local mission = Mission:new(xi.mission.log_id.WOTG, xi.mission.id.wotg.AFFAIRS_OF_STATE)
+
+mission.reward =
+{
+    keyItem = xi.ki.COUNT_BORELS_LETTER,
+    nextMission = { xi.mission.log_id.WOTG, xi.mission.id.wotg.BORNE_BY_THE_WIND },
+}
+
+mission.sections =
+{
+    -- Talk to these NPCs in either order:
+    -- Velda-Galda (K-9) in Windurst Waters (S)
+    -- Radford (H-6) in Bastok Markets (S)
+
+    -- When you finish both cutscenes, you will be given Count Borel's letter and begin the next mission.
+
+    -- 0. Speak to one of the nations first
+    {
+        check = function(player, currentMission, missionStatus, vars)
+            return currentMission == mission.missionId and missionStatus == 0
+        end,
+
+        [xi.zone.WINDURST_WATERS_S] =
+        {
+            ['Velda-Galda'] =
+            {
+                onTrigger = function(player, npc)
+                    -- TODO: What are these args from caps?
+                    return mission:progressEvent(178, 3, 2)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [178] = function(player, csid, option, npc)
+                    player:setCharVar("[WOTG12]WindurstFirst", 1)
```

Comment:
Similar to the above comment, setCharVar vs mission:setVar() and the references to the var below.

Reply:
👍 done